sys/riotboot/flashwrite.c

```diff
@@ -179,11 +177,11 @@ int riotboot_flashwrite_finish_raw(riotboot_flashwrite_t *state,
     int flashpage = flashpage_page((void *)slot_start);
     if (flashpage_write_and_verify(flashpage, firstpage) == FLASHPAGE_OK) {
         LOG_INFO(LOG_PREFIX "riotboot flashing completed successfully\n");
-        res = 0;
     }
     else {
         LOG_WARNING(LOG_PREFIX "re-flashing first block failed!\n");
+        return -1;
```

Comment:
Shouldn't the `LOG_WARNING` above become a `LOG_ERROR` ?

Reply:
Ok, changed it.